uint8 id
geometry_msgs/Point position
coverage_control2/PolygonWithHoles cell